src/utils.ts

```diff
@@ -110,7 +110,7 @@ function getCheckText(trivyStatus: number, dockleStatus: number): string {
   const trivyText = trivyHelper.getText(trivyStatus);
   let text = trivyText;
 
-  if (inputHelper.isCisChecksEnabled()) {
+  if (inputHelper.isRunQualityChecksEnabled()) {
```

Comment:
Suggestion: `isQualityChecksEnabled`

Reply:
Wanted to keep the function name consistent with the input variable name

Reply:
Both are fine. Leaving upto you.